server/sonar-web/src/main/js/types/types.d.ts

```diff
@@ -290,6 +291,12 @@ declare namespace T {
     name: string;
   }
 
+  // TODO
+  // Currently not being used, it has to be used.
```

Comment:
It should not be used. The group names are dynamic. The "sonar-administrators" is only one of system default group names.

Reply:
Ok. I will remove it.